frontend/packages/ceph-storage-plugin/src/components/ocs-install/types.ts

```diff
@@ -90,3 +90,8 @@ export type KMSConfigMap = {
   VAULT_CLIENT_KEY: string;
   VAULT_NAMESPACE: string;
 };
+
+export enum SupportedFeatures {
+  MULTUS = 'multus',
+  KMS = 'kms',
+}
```

Comment:
We need to include:
'arbiter' and 'flexibleScaling' too.

Reply:
Please include it while those features are getting added.